gradlew

```diff
@@ -86,8 +86,7 @@ done
 # shellcheck disable=SC2034
 APP_BASE_NAME=${0##*/}
 # Discard cd standard output in case $CDPATH is set (https://github.com/gradle/gradle/issues/25036)
-APP_HOME=$( cd -P "${APP_HOME:-./}" > /dev/null && printf '%s
-' "$PWD" ) || exit
```

Comment:
This change seems unrelated?

Reply:
This is the change that Grade made to the wrapper for 8.12, so it is related.

Reply:
Ah sorry, I wasn't aware of that.